ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Index.cshtml.cs

```diff
@@ -84,10 +94,36 @@ public async Task OnGetAsync()
 			}
 		}
 		
-		public bool UserHasEditCasePrivileges()
+		private bool UserHasEditCasePrivileges()
 		{
 			bool result = CaseModel.CreatedBy.Equals(User.Identity.Name, StringComparison.OrdinalIgnoreCase);
 			return result;
 		}
+
+		private async Task<bool> IsCaseClosed()
+		{
+			var statuses = await _statusCachedService.GetStatuses();
+			var caseStatus = statuses.FirstOrDefault(s => s.Urn.CompareTo(CaseModel.StatusUrn) == 0);
```

Comment:
Elijah, what if we use the _statusCacheService to fetch the closed status dto and then compare the urn ?